docs/markup-cicero.md

```diff
@@ -3,19 +3,45 @@ id: markup-cicero
 title: Overview
 ---
 
-## Principles
+## Markup Templating for Smart Contracts
 
-In order to make a clause executable, there needs to be a link to relate the template model to the natural language text that describes the legally enforceable clause. By inserting bindings to the template model into the natural language for the clause via the Accord Project markup language, we create what we call the template grammar. This determines what a syntactically valid clause may look like.
+In order to make a contract or clause executable, there needs to be a link to relate the Template Model to the natural language text of the contract or clause. A markup language is used to insert bindings to the Template Model into the Text. Together, the marked-up Text and Template Model is referred to as the Template Grammar. The Template Grammar defines a syntactically valid contract or clause.
 
 The Accord Project markup language is called CiceroMark. It supports:
-1. Rich text annotations based on [CommonMark](https://commonmark.org)
-2. Template variables and blocks for structured content _in the style of_ [Handlebars](https://handlebarsjs.com)
+1. Rich text annotations based on [CommonMark](https://commonmark.org) - a standard specification for Markdown
+2. Template variables and blocks for structured content _in the style of_ [Handlebars](https://handlebarsjs.com), including [Concerto](https://www.accordproject.org/projects/concerto/) models
 3. Formulas written as embedded [Ergo](https://docs.accordproject.org/docs/logic-ergo.html) expressions
 
-In a nutshell, CiceroMark is similar to Handlebars with two key differences:
+CiceroMark is similar to Handlebars with two key differences:
 - it is a template language for CommonMark rather than HTML
 - it uses Ergo as an expression language rather than JavaScript
 
+
+### An Extensible Framework for Contracts
+
+The CiceroMark markup language is used to turn contracts from static, unstructured, text into structured, machine-readable, documents. Contractual documents may take one of three evolutionary forms:
+
+1. Text: Natural language formats such as .docx or .pdf
```

Comment:
I find this somewhat confusing in this section. Possibly some of that could go in the front matter or in the "Concerto Model" section, but it would require review. The term `extensible` isn't a very clear concept in this context. I would keep it around but remove it from this PR.

Reply:
Will delete the entire `### An Extensible Framework for Contracts` section for this PR. The updates can be seen from https://github.com/peterhunn/techdocs/blob/master/docs/markup-cicero.md @peterhunn